### PR TITLE
Issue#434 add the possibility to change the preview image during the post updating

### PR DESCRIPTION
**Pull request by Valentyna-Chudik**
develop

## GitHub Board

[#434 Add the possibility to change the preview image during the post updating]( https://github.com/ita-social-projects/dokazovi-fe/issues/434)

**Story link**

[#319 Story](https://github.com/ita-social-projects/dokazovi-be/issues/319)
[#280 Story](https://github.com/ita-social-projects/dokazovi-be/issues/280)

## Summary of issue

- [x]  Add the possibility to change the preview image during the post updating

## Summary of change

- [x]  The possibility to change the preview image during the post updating added
- [x]  Preview image/video to previewPost added
- [x]  Duplicated files from "old" folder deleted

### CODE REVIEW

File: src/views/postUpdation/TextPostUpdation.tsx [hunk 1]
```diff
@@ -45,6 +49,9 @@ export const TextPostUpdation: React.FC<ITextPostUpdationProps> = ({
     post.origins,
   );
   const [htmlContent, setHtmlContent] = useState(post.content);
+  const [previewImageUrl, setPreviewImageUrl] = useState<string>(
```
Comment (michalenr) [inline src/views/postUpdation/TextPostUpdation.tsx:52]: I think it's not needed, typescript should infer type
```suggestion
  const [previewImageUrl, setPreviewImageUrl] = useState(
```

File: src/views/postUpdation/TextPostUpdation.tsx [hunk 2]
```diff
@@ -45,6 +49,9 @@ export const TextPostUpdation: React.FC<ITextPostUpdationProps> = ({
...
+    post.previewImageUrl as string,
```
Comment (michalenr) [inline src/views/postUpdation/TextPostUpdation.tsx:53]: What if it's undefined? Maybe better to do like this
```suggestion
    post.previewImageUrl ?? '',
```

File: src/views/postUpdation/TextPostUpdation.tsx [hunk 3]
```diff
@@ -105,12 +112,25 @@ export const TextPostUpdation: React.FC<ITextPostUpdationProps> = ({
     setSearchValue(authorFullName);
   };
 
+  const fileSelectorHandler = (
+    e: React.ChangeEvent<HTMLInputElement>,
+  ): void => {
+    getStringFromFile(e.target.files)
```
Comment (michalenr) [inline src/views/postUpdation/TextPostUpdation.tsx:118]: I think it would be cleaner with async await, but it's up to you

**Review by michalenr — APPROVED** [review]

File: src/views/postUpdation/TextPostUpdation.tsx (hunk 1 above)
Comment (Valentyna-Chudik) [reply src/views/postUpdation/TextPostUpdation.tsx]: Fixed

File: src/views/postUpdation/TextPostUpdation.tsx (hunk 2 above)
Comment (Valentyna-Chudik) [reply src/views/postUpdation/TextPostUpdation.tsx]: Fixed

**Review by AndriiKachkarov — APPROVED** [review]

**Pull request merged**